### DISCUSSION TITLE
Learning a function with uncertainty quantification

### DISCUSSION THREAD

**Discussion by AmosJoseph** [lululxvi/deepxde#820] [General]
In the Demo Learning a function with uncertainty quantification 

  Uncertainty:
    l2: 0.000304086
    
Excuse me, what's the meaning of  Uncertainty for l2？

Don't we focus on the Uncertainty of output?

**Comment by lululxvi** [comment]
The value itself doesn't make too much sense. It measures the strength of uncertainty of output.
